esp/services/esdl_svc_engine/esdl_binding.cpp

```diff
@@ -299,36 +350,20 @@ void EsdlServiceImpl::configureTargets(IPropertyTree *cfg, const char *service)
         IPropertyTree * customRequestTransform = nullptr;
         try
         {
-            customRequestTransform = target_cfg->queryPropTree("Method/xsdl:CustomRequestTransforms");
+            customRequestTransform = target_cfg->queryPropTree("xsdl:CustomRequestTransform");
         }
         catch (IPTreeException *e)
         {
-            m_custTrxCompileFail = true;
             StringBuffer msg;
             e->errorMessage(msg);
-            ERRLOG("Encountered error while fetching \"hpcc:Transforms\" from service \"%s\" ESDL configuration. Ensure a single set of transforms is provided:\n%s ", service, msg.str());
-            e->Release();
-        }
+            VStringBuffer errmsg("Encountered error while fetching \"xsdl:CustomRequestTransform\" from service \"%s\" ESDL configuration: %s ", service, msg.str());
 
-        try
-        {
-            if (customRequestTransform)
-                m_customRequestTransform.setown(new CEsdlCustomTransform(customRequestTransform));
-            m_custTrxCompileFail = false;
-        }
-        catch(IException* e)
-        {
-            m_custTrxCompileFail = true;
-            StringBuffer msg;
-            e->errorMessage(msg);
-            ERRLOG("Custom Request Transform could not be processed, will be ignored!!: \n\t%s", msg.str());
+            m_serviceLevelCrtFail = true;
+            ERRLOG("%s", errmsg.str());
             e->Release();
         }
-        catch (...)
-        {
-            m_custTrxCompileFail = true;
-            ERRLOG("Custom Request Transform could not be processed, will be ignored!!");
-        }
+
+        addServiceLevelRequestTransform(customRequestTransform);
```

Comment:
we don't explicitly throw from addServiceLevelRequestTransform, but are there any other potential exceptions we should catch?